packages/shared/src/graphql/squads.ts

```diff
@@ -205,6 +210,30 @@ export const SQUAD_QUERY = gql`
   ${SOURCE_BASE_FRAGMENT}
 `;
 
+export const MY_SQUADS_QUERY = gql`
+  query MySquads {
+    mySourceMemberships(first: 100, type: squad) {
```

Comment:
I think there's also a option to pass it as infinite maybe more failsafe (although don't think anyone but daphna will ever hit 100+ 🤣 )

Reply:
AFAIK, we limit to 100 by default on the API side anyway: https://github.com/dailydotdev/daily-api/blob/main/src/common/datePageGenerator.ts#L39
https://github.com/dailydotdev/daily-api/blob/main/src/schema/sources.ts#L878-L883

IMO, we cannot even show 100 items in the sidebar anyway, so having this paginated probably makes sense.